packages/cli/src/__tests__/index.test.ts

```
@@ -0,0 +1,29 @@
+/**
+ * CLI Tests
+ * Basic tests for CLI functionality
+ */
+
+describe('CLI', () => {
+  it('should be defined', () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)